diff --git a/node_modules/@expo/ui/ios/Button/Button.swift b/node_modules/@expo/ui/ios/Button/Button.swift
index ebe5aaa..86e5c22 100644
--- a/node_modules/@expo/ui/ios/Button/Button.swift
+++ b/node_modules/@expo/ui/ios/Button/Button.swift
@@ -39,7 +39,12 @@ struct Button: ExpoSwiftUI.View {
     .if(props.variant == .borderedProminent, {
       $0.buttonStyle(.borderedProminent)
     })
-    #if !os(tvOS)
+
+    #if os(tvOS)
+    .if(props.variant == .card, {
+      $0.buttonStyle(.card)
+    })
+    #else
     .if(props.variant == .borderless, {
       $0.buttonStyle(.borderless)
     })
@@ -60,7 +65,7 @@ struct Button: ExpoSwiftUI.View {
     })
     #endif
 
-    if #available(iOS 26.0, *) {
+    if #available(iOS 26.0, tvOS 26.0, *) {
       #if compiler(>=6.2) // Xcode 26
       switch props.variant {
       case .glass:
